brain_brew/representation/yaml/note_model_template.py

```diff
@@ -23,7 +23,7 @@
 HTML_FILE = AnkiField("html_file")
 BROWSER_HTML_FILE = AnkiField("browser_html_file", default_value=None)
 
-html_separator_regex = r'[\n]{1,}[-]{1,}[\n]{1,}'
+html_separator_regex = r'[(\r\n|\r|\n)]{1,}[-]{1,}[(\r\n|\r|\n)]{1,}'
```

Comment:
```suggestion
html_separator_regex = r'(\r\n|\r|\n){1,}[-]{1,}(\r\n|\r|\n){1,}'
```

Shouldn't we remove the square brackets here, replacing the with braces for a capture group? Playing around with this on regex101.com it seems that `[(\r\n|\r|\n)]` is simple matching any individual character (as square brackets do in regex) including opening and closing braces 😅 

![image](https://github.com/ohare93/brain-brew/assets/1471350/933bb06e-3bf6-46a8-9d3b-5b414a4a1d6e)

Removing the square brackets seems to achieve the desired effect 😄

Reply:
Oh, seems like I overlooked that. You are right of course. However, the groups need to be declared non-capturing to not get more than 1 match. I added these changes accordingly.

Reply:
I haven't tested and I haven't used regexes for a while, but I believe that capturing vs. non-capturing only affects whether the contents of the group can be accessed later.